lexicons/com/atproto/admin/getAccountInfos.json

```diff
@@ -0,0 +1,36 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.admin.getAccountInfos",
```

Comment:
this is a batch version of `getAccountInfo` (implemented on PDS) - think there's a decent argument we just deprecate the non-batch version

The basic idea here is to go to all relevant services (pds & appview) for all the info they have on a user & then collate in the ozone backend